client/post-editor/editor-term-selector/index.jsx

```diff
@@ -82,6 +84,7 @@ export default connect(
 		return {
 			postType: getEditedPostValue( state, siteId, getEditorPostId( state ), 'type' ),
 			postTerms: getEditedPostValue( state, siteId, postId, 'terms' ),
+			canEditTerms: canCurrentUser( state, siteId, 'manage_categories' ),
```

Comment:
Not certain I follow how I could implement a similar strategy here.  Wouldn't it just be best to get `edit_terms` added into the API response and refactor then?